src/components/UnreadActionIndicator.js

```diff
@@ -1,29 +1,19 @@
 import React from 'react';
-import {Animated, View} from 'react-native';
-import PropTypes from 'prop-types';
-import styles, {getOpacityStyle} from '../styles/styles';
+import {View} from 'react-native';
+import styles from '../styles/styles';
 import Text from './Text';
 
-const propTypes = {
-    // Animated opacity
-    // eslint-disable-next-line react/forbid-prop-types
-    animatedOpacity: PropTypes.object.isRequired,
-};
-
-const UnreadActionIndicator = props => (
-    <Animated.View style={[
+const UnreadActionIndicator = () => (
+    <View style={[
```

Comment:
Small detail, but I think this can be cleaned up to just:

```suggestion
    <View style={styles.unreadIndicatorContainer}>
```

Reply:
Done!